### PR TITLE
Fix llm test

### PR DESCRIPTION
**Pull request by keepingitneil**

### CODE REVIEW

**Review by theomonnom — APPROVED** [review]

File: tests/test_llm.py [hunk 1]
```diff
@@ -159,6 +159,12 @@ async def test_cancelled_calls():
     stream = await _request_fnc_call(
         llm, "Turn off the lights in the Theo's bedroom", fnc_ctx
     )
+    stream.execute_functions()
+
+    # TODO (?): need to wait for the task to be scheduled but this might be
+    # a design flaw. Users would not be able to respond to cancelled tasks
+    # if they are never started.
```
Comment (theomonnom) [inline tests/test_llm.py:166]: I think we should be able to get the specific tasks handle when iterating over the stream

File: tests/test_llm.py [hunk 2]
```diff
@@ -159,6 +159,12 @@ async def test_cancelled_calls():
...
+    await asyncio.sleep(0.1)
```
Comment (theomonnom) [inline tests/test_llm.py:167]:
```suggestion
    await asyncio.sleep(0)
```

File: tests/test_llm.py (hunk 1 above)
Comment (keepingitneil) [reply tests/test_llm.py:166]: Oh yes true, maybe my comment can be updated. Really what it means is that it's possible for tasks to never start but idk if it's an issue 🤔

**Comment by davidzhao** [comment]
nice! back to greeeen

**Pull request merged**